<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingUserForm</class>
 <widget class="QWidget" name="QSettingUserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_settinguser_pgup">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_settinguser_pgdn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_settinguser_delete">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_settinguser_add">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_settinguser_save">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settinguser_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settinguser_passwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settinguser_passwdcomfire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settinguser_people">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settinguser_time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settinguser_note">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_passwd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_passwdcomfrie">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_people">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_time">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_note">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tb_settinguser_list">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settinguser_quanxian">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_settinguser_quanxian">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_settinguser_change">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_usertitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDefaultLogin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
